pebblo/app/service/local_ui_service.py

```diff
@@ -52,28 +51,37 @@ def get_all_apps_details():
                     logger.debug(f"metadata.json path {app_path}")
                     app_json = read_json_file(app_path)
 
-                    # Skip app if unable to fetch load id details
-                    if not app_json and not app_json.get("load_ids"):
+                    if not app_json:
+                        # Unable to find json file
+                        logger.debug(
+                            f"Metadata file ({CacheDir.METADATA_FILE_PATH.value}) not found for app: {app_dir}."
+                        )
                         logger.warning(
-                            f"Error: Unable to fetch loadId details for {app_dir} app"
+                            f"Skipping app '{app_dir}' due to missing or invalid file"
```

Comment:
The same error message is logged at multiple places. It could be more descriptive/specific

Reply:
Warning message are for user and debug message are for developer.
Warning messages are kept simple for user to understand.